Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Processes/TestToolProcess.cs

```diff
@@ -37,6 +39,17 @@ public static TestToolProcess Startup(RunCommandSettings settings, CancellationT
     {
         var builder = WebApplication.CreateBuilder();
 
+        builder.Services.AddSingleton(typeof(Assembly), typeof(ConfigurationSetup).Assembly);
```

Comment:
nit: Since this is a copy of the same code in `ApiGatewayEmulatorProcess` can you put it in a utility method?